lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -253,10 +223,14 @@ public function render_default_login_redirection( $args = array() ) {
 	 * Render form field and description for the `lock_connections` option.
 	 * IMPORTANT: Internal callback use only, do not call this function directly!
 	 *
+	 * @deprecated - 3.10.0, moved to Features section.
```

Comment:
Deprecating here, a deprecation PR for methods marked as internal seemed overkill.